packages/@jsii/go-runtime-test/build-tools/gen-calc.ts

```diff
@@ -1,12 +1,10 @@
 #!/usr/bin/env npx ts-node
```

Comment:
Since `jsii-calc` is not need in `go-runtime` now, this file was copied over here and adapted.